### ISSUE TITLE
Debug with chrome error after upgrading to 0.17

### ISSUE THREAD

**Issue by AdamGorkoz** [facebook/react-native#4916]
Getting this error in the console : 
Uncaught TypeError: Cannot read property 'apply' of undefined

Please help.

**Comment by jozn** [comment]
+1 the same problem.
Uncaught TypeError: Cannot read property 'apply' of undefined .   
source: debuggerWorker.js:39

**Comment by jozn** [comment]
+1
Uncaught TypeError: Cannot read property 'apply' of undefinedonmessage @ debuggerWorker.js:39
debuggerWorker.js39
OS: windows 8

**Comment by satya164** [comment]
Do you have a stack trace or something? Without a stack trace it's impossible to figure out where this is happening.

**Comment by turka50** [comment]
@satya164  This is exactly where it happens , I'm using Android btw : 

![image](https://cloud.githubusercontent.com/assets/10440278/11972740/2c8a20d0-a959-11e5-8b54-e78ae35e7200.png)

There isn't a stack really , just one line of error : 
![image](https://cloud.githubusercontent.com/assets/10440278/11972750/49db99e8-a959-11e5-8d10-d29b4125419c.png)

**Comment by satya164** [comment]
Might be related to #4881

**Comment by dgilperez** [comment]
Same here.

**Comment by satya164** [comment]
Can you try with the current master and see if the issue still persists?

**Comment by drtangible** [comment]
Also seeing this on master, only when debugging in Chrome in Android.

Logging what's missing in `debuggerWorker.js`:

```diff
    try {
      if (typeof __fbBatchedBridge === 'object') {
+      console.log("Looking for", object.method, "in", __fbBatchedBridge);
        returnValue = __fbBatchedBridge[object.method].apply(null, object.arguments);
      }
    } finally {
```

Appears it's unable to find `executeJSCall` on `__fbBatchedBridge`.

<img width="651" alt="screenshot 2015-12-28 11 45 50" src="https://cloud.githubusercontent.com/assets/169912/12022743/612cde1a-ad58-11e5-8f1e-5ae88dc2882d.png">

**Comment by drtangible** [comment]
Able to work around this by reverting a piece of the changeset from 8d397b4c, specifically the change in `debuggerWorker.js` (3017fb57). @sebmarkbage, not sure how to debug this further, nor why this may only be popping up in Android w/Chrome debugging. Any thoughts?

**Comment by sebmarkbage** [comment]
I think that is an issue with the binary native bundle being out of sync with the JS bundle and packager. Try redeploying the app bundle that you're testing.

**Comment by drtangible** [comment]
@sebmarkbage :+1:, looks like I forgot to run `react-native upgrade` while upgrading, and that seems to resolve things. Thanks!

**Comment by Hyperonic** [comment]
I'm still getting this error even after upgrading

**Comment by haydenth** [comment]
I'm still getting this error over here too - 

![screen shot 2016-01-09 at 11 00 09 am](https://cloud.githubusercontent.com/assets/1082756/12217244/30c79532-b6c0-11e5-8124-c82a141f6bcf.png)

![screen shot 2016-01-09 at 11 00 12 am](https://cloud.githubusercontent.com/assets/1082756/12217247/34c0287a-b6c0-11e5-89de-6cbf39c3d061.png)

**Comment by jforaker** [comment]
I get the error too in Android, only after selecting "Debug in Chrome":

```Uncaught TypeError: Cannot read property 'apply' of undefined
onmessage @ debuggerWorker.js:39```

![image](https://cloud.githubusercontent.com/assets/2042031/12217734/71cbd748-b6d7-11e5-851e-50685f4c1851.png)



Seems "executeJSCall" is not part of the `__fbBatchedBridge` object



![image](https://cloud.githubusercontent.com/assets/2042031/12217739/940d34fa-b6d7-11e5-9092-cede86e3e442.png)


STR:
1. `npm install react-native`  (which was  "^0.18.0-rc")
2. `react-native upgrade`
3. Run and debug in chrome > see error in console and white screen on device

**Comment by Overload119** [comment]
+1

**Comment by haydenth** [comment]
Still getting this error on 0.18 - debuggerWorker.js, line 39 now. "Cannot read property 'apply' of undefined"

**Comment by amit-Newput** [comment]
+1
Getting this error, when debugging in Chrome in Android

**Comment by haydenth** [comment]
Still an issue on Android 0.19

**Comment by haydenth** [comment]
Figured it out. Upgraded to 0.19 and ran react-native upgrade. I let it overwrite all my android config files: app/build.gradle, settings file, and the MainActivity.java. Then I went through and basically moved the code I had from before back in here. Uninstalled from phone and did a fresh build. Works now.

**Comment by dabit3** [comment]
I had this error after changing versions to 0.20. This goes away when you run react-native upgrade .

**Comment by liu402883451** [comment]
+1
